rasa/core/policies/ted_policy.py

```diff
@@ -994,20 +1000,32 @@ def load(
         model_storage: ModelStorage,
         resource: Resource,
         execution_context: ExecutionContext,
+        rule_only_data: Optional[Dict[Text, Any]] = None,
```

Comment:
Hm, this is ok because we inspect all methods including the `load` and hence will notice that it is `rule_only_data` that needs to be provided and not a kwarg, right?

Reply:
🤦🏻 This actually only works during training and not in this case 🙈 Will fix it

Reply:
I thought I'm clever with the `load` thing but that actually only works during training. Passing it via `predict_action_probabilities` now